### PR TITLE
[OSPP][PIR] support dropout and squeeze

### PR DESCRIPTION
**Pull request by qzylalala**
We support dropout and squeeze operation, and we can export models in `paddle.vision.models` :
* resnet50
* mobilenet_v2
* shufflenet_v2
* densenet121
* inception_v3
* squeezenet

### CODE REVIEW

File: paddle2onnx/parser/pir_parser.cc
Comment (0x3878f) [inline paddle2onnx/parser/pir_parser.cc:1]: 删掉这些打印信息吧

Comment (0x3878f) [inline paddle2onnx/parser/pir_parser.cc:1]: 删掉这些打印信息吧

**Review by 0x3878f — CHANGES_REQUESTED** [review]
LGTM

**Review by 0x3878f — CHANGES_REQUESTED** [review]
LGTM

File: paddle2onnx/parser/pir_parser.cc [hunk 1]
```diff
@@ -229,6 +229,11 @@ namespace paddle2onnx {
           // paddle::dialect::IsLegacyOp(op_name));
           false);
       name = GetOpArgName(op_id, name);
+      P2OLogger() << "Op name : " << op_name << std::endl;
+      for (auto [k, v] : yaml_parser.InputName2Id()) {
+        P2OLogger() << "  Arg name : " << k << " , idx : " << v << std::endl;
+      }
+      P2OLogger() << "  Arg name : " << name << std::endl;
```
Comment (risemeup1) [inline paddle2onnx/parser/pir_parser.cc]: 这些不必要的调试信息尽量删除掉

Comment (qzylalala) [reply paddle2onnx/parser/pir_parser.cc]: Done

**Review by 0x3878f — COMMENTED** [review]
LGTM

**Pull request merged**